src/state/edificio.js

```diff
@@ -0,0 +1,12 @@
+import { selector, selectorFamily } from 'recoil';
+import { getData } from '../helpers/fetchApi';
+
+export const todosLosEdificios = selector({
+  key: 'todosLosEdificios',
+  get: async () => (await getData('edificios')).data,
+});
+
+export const edificioPorId = selectorFamily({
+  key: 'edificioPorId',
+  get: (id) => async () => await getData(`edificios/${id}`),
```

Comment:
En este caso, se puede omitir el `async/await` y va a funcionar igual porque devuelve una Promise. Cuestión de gustos nomás.

```suggestion
  get: (id) => () => getData(`edificios/${id}`),
```